src/storage/database_sink.rs

```diff
@@ -154,6 +154,8 @@ impl<'s> DatabaseSink<'s> {
             ));
         }
 
+        fields.push("geom geometry(Point)".to_owned());
+
```

Comment:
In the test `schema_creation_leads_to_required_columns_in_unit_table` we call the function `retrieve_ordered_table_column_names` to check that every column is there. There, we add the surrogate key explicitly. Why doesn't this need to be altered when we add a new `geom` column? Shouldn't it crash since there is a new column?

Reply:
it actually failed for me but I fixed it now dbeeec65b9484ae89be714f6a9106092ada0a7ec